opm/grid/GraphOfGridWrappers.hpp

```diff
@@ -180,6 +181,7 @@ void extendExportAndImportLists(const GraphOfGrid<Dune::CpGrid>& gog,
                           std::vector<std::tuple<int,int,char>>& exportList,
                       std::vector<std::tuple<int,int,char,int>>& importList,
                                          const std::vector<int>& gIDtoRank={});
+#endif // HAVE_MPI
```

Comment:
You probably should end the namespace in the if statement or move it to outside in line 133-134?